.gitlab-ci.yml

```diff
@@ -229,6 +229,7 @@ deploy-staging:
     - yarn
     - BUILD_MODE=canary yarn build:bundle
     - ./scripts/deploy.sh staging staging
+    - BUILD_MODE=canary node ./scripts/upload-source-maps.js datad0g.com,datadoghq.com staging
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
You can avoid duplicating `BUILD_MODE=canary` by setting it before other scripts:
```suggestion
    - BUILD_MODE=canary 
    - yarn build:bundle
    - ./scripts/deploy.sh staging staging
    - node ./scripts/upload-source-maps.js datad0g.com,datadoghq.com staging
```